onboarding-presentation/src/main/java/com/teamhy2/onboarding/OnboardingScreen.kt

```diff
@@ -23,31 +26,39 @@ import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
-import androidx.compose.ui.viewinterop.AndroidView
-import com.google.android.gms.common.SignInButton
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.teamhy2.core.auth.GoogleSignInButton
 import com.teamhy2.designsystem.ui.theme.BackgroundBlack
 import com.teamhy2.designsystem.ui.theme.Gray600
 import com.teamhy2.designsystem.ui.theme.HY2Theme
 import com.teamhy2.onboarding.presentation.R
 
 @Composable
 fun OnboardingRoute(
-    onGoogleLoginClick: () -> Unit,
-    onGoogleLoginDone: () -> Unit,
+    onGuestSignedIn: () -> Unit,
```

Comment:
```suggestion
    onPublicUserSignedIn: () -> Unit,
```
아직 가입하지 않은 유저를 일관성 있게 `PublicUser`로 네이밍을 변경하는 것은 어떤가요

Reply:
Guest도 사실 좋습니다! 비로그인으로 서비스를 이용할 수 있는 서비스의 느낌이 나서 제안해보았습니다!

Reply:
사실 가입을 하고 로그인을 하는 플로우가 일반적인데ㅠ
정반대라 네이밍을 하는 것도 조금 어렵게 느껴지는 군요ㅠ
지금은 게스트가 그나마 잘어울리는 거 같아요. 로그인은 했지만, 고객은 아닌...
일단은 현행 유지하고 추후, 게스트 로그인을 저희가 구현한다면 그때 다시 네이밍 변경을 고려해보겠습니다!

Reply:
네 말씀해주신 내용 이해하였고 확인하였습니다!